README.md

```diff
@@ -25,199 +25,69 @@ The purpose of Amber is not to create yet another framework, but to take advanta
 
-## Read the Docs
-
-Documentation https://docs.amberframework.org
+## Documentation
 
+Read Amber documentation on https://docs.amberframework.org
 
 ## Benchmarks
 
-Latest Results **968,824.35 requests per second: 32 cores at 2.7Ghz**
-
-```bash
-ubuntu@ip-172-31-0-70:~/bench⟫ wrk -d 60 -t 20 -c 1015 http://localhost:3000
-Running 1m test @ http://localhost:3000
-  20 threads and 1015 connections
-  Thread Stats   Avg      Stdev     Max   +/- Stdev
-    Latency     1.86ms    2.88ms  56.61ms   87.54%
-    Req/Sec    48.73k     6.01k   88.40k    68.28%
-  58225168 requests in 1.00m, 4.01GB read
-Requests/sec: 968824.35
-Transfer/sec:     68.37MB
-```
-
-> Disclaimer: We share these benchmark results with the understanding they may vary depending on configuration and environment settings. By no means are we making any comparison claims in regards to other web application frameworks.
-
-## Installation
-
-#### Linux
-
-Ensure you have the necessary dependencies:
+[Techempower Framework Benchmarks - Round 15 (2018-02-14)](https://www.techempower.com/benchmarks/#section=data-r15&hw=ph&test=fortune&f=zik073-zik0zj-zik0zj-zik0zj-zhxjwf-zik0zj-gnbmym-cn3)
 
-- `git`: Use your platform specific package manager to install `git`
-- `crystal`: Follow the instructions to get `crystal` on this page: <https://crystal-lang.org/docs/installation/index.html>
+Fortunes test comparing [Amber](https://amberframework.org/), [Kemal](https://kemalcr.com/), [Rails](https://rubyonrails.org/), [Phoenix](https://phoenixframework.org/), and [Hanami](https://hanamirb.org/):
 
-##### For Debian & Ubuntu
+[![framework-benchmark](https://github.com/amberframework/site-assets/raw/master/images/benchmarks-fortunes.png "TFB Fortunes test for Crystal, Elixir and Ruby frameworks")](https://www.techempower.com/benchmarks/#section=data-r15&hw=ph&test=fortune&f=zik073-zik0zj-zik0zj-zik0zj-zhxjwf-zik0zj-gnbmym-cn3)
 
-These are necessary to compile the CLI:
+## Installation & Usage
 
-- apt-get install build-essential libreadline-dev libsqlite3-dev libpq-dev libmysqlclient-dev libssl-dev libyaml-dev
+#### macOS
 
-<!-- WIP: ##### For RedHat & CentOS
-- `sudo yum groupinstall 'Development Tools' `
-- `sudo yum install readline-devel sqlite-devel openssl-devel libyaml-devel gc-devel libevent-devel` -->
-
-Once you have these dependencies, you can build `amber` from source:
-
-
-```shellsession
-$ git clone git@github.com:amberframework/amber.git
-$ cd amber
-$ make
-$ make install
 ```
-
-##### For ArchLinux & derivates
-- Install the CLI from [AUR package](https://aur.archlinux.org/packages/amber/). Dependencies are automatically installed.
-- `yaourt -S amber`
-
-You should now be able to run `amber` in the command line.
-
-#### Mac OS X
-
-Best way to get `amber` on Mac OS X is via Homebrew:
-
-```shellsession
-$ brew install amberframework/amber/amber
+brew tap amberframework/amber
+brew install amber
```

Comment:
@elorest - is the `tap` still necessary or will a simple `brew install amber` work now?

Reply:
`brew install amber` works

Reply:
Done! :+1: